MapleServer2/Database/DatabaseManager.cs

```diff
@@ -10,7 +10,7 @@ namespace MapleServer2.Database;
 public static class DatabaseManager
 {
     private static readonly ILogger Logger = Log.Logger.ForContext(typeof(DatabaseManager));
-    private static readonly int MIN_MYSQL_VERSION = 8;
+    private static readonly Tuple<int, int> MIN_MYSQL_VERSION = new(8, 0); // Primary support version of MySQL (major version, minor version)
```

Comment:
I recommend using this system since its easier to read and you can have name for the items. It's still a Tuple.
Please change all Tuple<int, int> to this style (int variableName, int variableName2)

```suggestion
    private static readonly (int Major, int Minor) MIN_MYSQL_VERSION = new(8, 0); // Primary support version of MySQL (major version, minor version)
```